transcripts/5-21-2020.md

```diff
@@ -0,0 +1,139 @@
+# 5/21/2020 ESLint TSC Meeting Transcript
+
+**kaicataldo:** Hello 👋
+**nzakas:** Hi there! Welcome to our Discord channel for TSC meetings!
+**btmills:** 👋 smells like fresh paint in here!
+**nzakas:** Hehe
+**nzakas:** Looks like <@!235597315367239700> can't join us, so we're ready to get started.
```

Comment:
Might be good to replace these user id's with their Discord username. I'll take a look at that tomorrow.